scripts/clone-cloud-components.sh

```diff
@@ -2,7 +2,7 @@
 
 rm -rf ./tmp/openfaas-cloud
 
-git clone https://github.com/openfaas/openfaas-cloud ./tmp/openfaas-cloud
+git clone https://github.com/openfaas/openfaas-cloud --depth 1 ./tmp/openfaas-cloud
```

Comment:
You can checkout a branch directly apparently with `--branch`

Not sure about SHAs though